web/source/osk/visualKeyboard.ts

```diff
@@ -1317,8 +1317,8 @@ namespace com.keyman.osk {
         // Find the actual key element.
         let childNode = keySquare.firstChild ? keySquare.firstChild as HTMLElement: keySquare;
 
-        if(childNode.className !== undefined  && (childNode.className.indexOf('key-hidden') >= 0)
-             || (childNode.className.indexOf('key-blank') >= 0)){
+        if(childNode.className !== undefined  && ((childNode.className.indexOf('key-hidden') >= 0)
+             || (childNode.className.indexOf('key-blank') >= 0))){
```

Comment:
```suggestion
        if(childNode.className !== undefined
            && (childNode.className.indexOf('key-hidden') >= 0
             || childNode.className.indexOf('key-blank') >= 0)) {
```

Can reduce parens rather than increasing?